packages/kbn-apm-synthtrace-client/src/lib/utils/generate_id.ts

```diff
@@ -7,10 +7,15 @@
  */
 
 let seq = 0;
+const pid = String(process.pid);
 
 function generateId(seed?: string, length: number = 32) {
-  const str = seed ?? String(seq++);
-  return str.padStart(length, '0');
+  if (seed) {
+    return seed.padStart(length, '0');
```

Comment:
to not mess up with `error.grouping_key` the id is not concatenated with `pid`

Reply:
Hmm it gets a little muddy here. I wonder if we should remove `seed` from `generateShortId()` and `generateLongId` and expose `generateIdWithSeed()`. That at least makes it a little more explicit.

Reply:
good idea!